examples/step-27/step-27.cc

```diff
@@ -605,7 +605,7 @@ namespace Step27
 
         setup_system();
 
-        std::cout << "   Number of active cells:       "
+        std::cout << "   Number of active cells      : "
```

Comment:
I have to say that I'm not the biggest fan of this placement of the colon -- but it's not worth arguing about.

Reply:
Ah, I see what you mean -- I would have just removed the space on all three lines. But who cares :-)